charts/rancher-external-dns/v0.0.3/README.md

```diff
@@ -0,0 +1,132 @@
+# external-dns
+
+## Chart Details
+
+This chart will do the following:
+
+* Create a deployment of [external-dns] within your Kubernetes Cluster.
+
+Currently this uses the [Zalando] hosted container, if this is a concern follow the steps in the [external-dns] documentation to compile the binary and make a container. Where the chart pulls the image from is fully configurable.
+
```

Comment:
@StrongMonkey maybe we should update the ReadMe to mention the source of the image is the official release from https://github.com/kubernetes-sigs/external-dns

Reply:
:+1:

Reply:
@prachidamle Updated.